### PR TITLE
adding run_conode to cothority_template

### PR DESCRIPTION
**Pull request by ineiti**
When starting out for a project, it should be possible to run a conode simply from the cothority_template directory.

### CODE REVIEW

File: conode/run_conode.sh [hunk 1]
```diff
@@ -0,0 +1,345 @@
+#!/usr/bin/env bash
```
Comment (nikkolasg) [inline conode/run_conode.sh:1]: Is there no way to use the one defined in onet or cothority somewhere ? Instead of duplicating the code.

Comment (ineiti) [reply conode/run_conode.sh:1]: You'd have to be able to import files after the compilation - I don't know how. This is one reason I put most of the functionality in `onet/app`, so the `conode.go` is very lightweight. I corrected the current conode.go and really do use the `onet/app`, which was not the case before.

Thanks for keeping track of my changes ;)

Comment (nikkolasg) [reply conode/run_conode.sh:1]: > You'd have to be able to import files after the compilation

I was merely talking about the bash script. Even a simple  `git clone gopkg.in/dedis/onet.v1/app/run_conode.sh && ./run_conode.sh $@` would be better no ?

Comment (ineiti) [reply conode/run_conode.sh:1]: Now I see the light. True, but for the moment I have some hard-coded paths in there :( So first I'd need to change these... Is it worth it?

**Pull request merged**